src/Simple.php

```diff
@@ -750,6 +758,12 @@ public function stop($reason)
         throw new ApplicationStoppedException($reason);
     }
 
```

Comment:
Some documention would be nice, for example when it should be used instead of the `stop()` method.